build/amis/README.md

```diff
@@ -125,5 +125,5 @@ Connect remotely to an instance created from the image and run the Node Conforma
 wget https://dl.k8s.io/$(< /etc/kubernetes_community_ami_version)/kubernetes-test.tar.gz
 tar -zxvf kubernetes-test.tar.gz kubernetes/platforms/linux/amd64
 cd kubernetes/platforms/linux/amd64
-sudo ./ginkgo --nodes=8 --flakeAttempts=2 --focus="\[Conformance\]" --skip="\[Flaky\]|\[Serial\]|\[sig-network\]|Container Lifecycle Hook" ./e2e_node.test -- --k8s-bin-dir=/usr/bin
-```
\ No newline at end of file
+sudo ./ginkgo --nodes=1 --flakeAttempts=2 --focus="\[Conformance\]" --skip="\[Flaky\]|\[Serial\]|\[sig-network\]|Container Lifecycle Hook" ./e2e_node.test -- --k8s-bin-dir=/usr/bin
```

Comment:
Why was this changed?

Reply:
this doesn't have to change. I haven't been able to get to this to work at all. I thought it was running when I sumbitted the PR but it ended up running but not able to schedule any pods. Need to configure *something* but i'm still debugging.